scripts/fuzz_opt.py

```diff
@@ -35,6 +35,7 @@
 
 # parameters
 
+# TODO: switch to --hybrid when that becomes the most common mode for users
 TYPE_SYSTEM_FLAG = '--nominal'
```

Comment:
Can we make the fuzzer randomize this so we get coverage of both modes right away? That would be useful in the longer term as well if we keep the `--nominal` option around.